vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler.go

```diff
@@ -157,3 +170,19 @@ func validateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
 
 	return nil
 }
+
+func validateResource(name corev1.ResourceName, val apires.Quantity) error {
```

Comment:
We don't share any code between CPU and memory. I think it would be simpler to have two functions (for example `validateCPUResolution` and `validateMemoryResolution`)

Reply:
okay